src/templates/sidebar/combat-tracker.hbs

```diff
@@ -1,124 +1,147 @@
+{{! Defines a partial that is used below to iterate over various button contexts}}
+{{#*inline "buttonPartial"}}
+	{{#each buttons}}
+		<a class="combatant-control {{#if (lookup ../combatant property)}}active{{/if}}" id={{id}} data-event={{eventName}} data-property="{{property}}" data-tooltip="{{localize tooltip}}">
+			<i class="fas {{icon}}"></i></a>
+	{{/each}}
+{{/inline}}
```

Comment:
Small thought: Wouldn't this be better placed into the `./src/module/handlebars.js` file ?

Reply:
Resolve: better design as used nowhere else